### ISSUE TITLE
No green box. Console errors

### ISSUE THREAD

**Issue by ieatpuppies** [chrisgoringe/cg-image-picker#42]
Used the previous versions in my workflows with no issue. This latest update is causing issues. Comfyui is up to date.

Using the basic workflow in docs:

Right clicking a chooser image gives me nothing. No green box, no menu, nothing. Have to fully kill Comfyui and restart since I can't cancel the workflow. Noticed some errors related to Nestednodebuilder in my console, so I tried removing that. Still no green box, but now I have the bog standard right click menu at least.

Remove and reinstall image chooser. No change. Reinstall nestednodebuilder. Back to no right click menu. Any ideas?

Here's the console dump when i right click: 

Uncaught TypeError: cyclic object value
cloneObject http://127.0.0.1:8188/lib/litegraph.core.js:599
serializeWorkflow http://127.0.0.1:8188/extensions/ComfyUI_NestedNodeBuilder/nestedNode.js:26
getExtraMenuOptions http://127.0.0.1:8188/extensions/ComfyUI_NestedNodeBuilder/nodeMenu.js:235
getNodeMenuOptions http://127.0.0.1:8188/lib/litegraph.core.js:13180
processContextMenu http://127.0.0.1:8188/lib/litegraph.core.js:13302
processMouseDown http://127.0.0.1:8188/lib/litegraph.core.js:6269
processMouseDown http://127.0.0.1:8188/scripts/app.js:846
litegraph.core.js:599:37

**Comment by chrisgoringe** [comment]
Quick q's:
- did you look at the error console when nestednodebuilder was removed?
- could you post a workflow?
- what other custom nodes are installed?

**Comment by chrisgoringe** [comment]
Update 2.5.3 *may* have fixed this.

**Comment by ieatpuppies** [comment]
To answer your questions. 

Tried the update, still didn't work. I've since installed a second instance of Comfyui and omitted all of the custom nodes I wasn't really using. That second instance has no issues with image chooser. I was trying to pinpoint which node was causing the issue, but removing stable-wildcards deleted my custom_nodes directory entirely, so I got annoyed and just swapped to the new install entirely. 

To answer your questions:

1. There were no errors in the log when nestednodebuilder was removed. However, the chooser simply failed to function. 
2. Workflow tested was the basic one you uploaded in docs.
3. 
A fresh install with the following nodes works correctly:
Manager, impact pack, controlnet auxiliary preprocessors, efficiency nodes, was node suite, freeu_advanced, pythongosssss custom scripts, ultimatesdupscale, comfyui_ipadapter_plus, wlsh_nodes, rgthree's comfyui 

The install that was failing had the following nodes: 
Manager, Impact Pack, **_Inspire Pack_**, controlnet Auxiliary Preprocessors, **_CLIPSeg_**, Efficiency Nodes, **_Derfuu_comfyui_moddednodes_**, WAS node suite, **_masquerade nodes_**, freeu_advanced, pythongosssss custom scripts, UltimateSDUpscale, **_ComfyUI-Bmad-DirtyUndoRedo_**, **_SeargeSDXL_**, **_Simple Math_**, Comfyui_ipadapter_plus, **_comfymath_**, wlsh_nodes, **_reactor node 0.1.0 for comfyui_**, rgthree's comfyui nodes, **_bsz-cui-extras_**, **_use everywhere_**, **_save image extended_**, **_comfyui-stable-wildcards_**

Everything in bold italics is the delta between the working and non-working installs.

**Comment by chrisgoringe** [comment]
thanks - great detail. I'll try adding the bold italics packs to a clean install and approach from that side!

**Comment by chrisgoringe** [comment]
OK - looks like the culprit is DirtyUndoRedo. And the reason is that they capture mouse clicks and don't pass them on.

Can you try with everything except that? I'll raise an issue on their github.

**Comment by chrisgoringe** [comment]
I've raised an issue https://github.com/bmad4ever/ComfyUI-Bmad-DirtyUndoRedo/issues/2 and even done a PR with the required fix for them https://github.com/bmad4ever/ComfyUI-Bmad-DirtyUndoRedo/pull/3

**Comment by ieatpuppies** [comment]
Ok, attempted with all of those nodes minus the DirtyUndoRedo. No green box, no option to progress image, right click menu available, no messaging in the console. 

So I removed bsz-cui-extras, derfuu, simple math, comfymath, and still nothing. So I just deleted that install of Comfy. 

Then I tried using all those bold/italics nodes on a new Comfy install, and everything worked fine. Added DirtyUndoRedo, back to not working. Removed DirtyUndoRedo again, still does not work. 

So it looks like installing DirtyUndoRedo creates the issue, and it does not go away even if you uninstall it. 

Which is easy enough to avoid because i wasn't using it anyway. Hopefully they respond to the issue so other folks don't hit the same trap. 

Thanks for helping track down the problem.
